style.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background-color: #fff;
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: 0.5px;
+    height: 100%;
+  }
+  
+  header {
+    display: flex;
+    flex-direction: row;
+  }
+  
+  #brand {
+    margin-left: 20px;
+  }
+  .youtube {
+    height: auto;
+    width: 30px;
+  }
+  
+  .form-inline {
+    display: flex;
+    align-items: center;
+    margin-left: 470px;
+    /* margin-right: -200px; */
+  }
+  
+  .input-group {
+    width: 100%;
+    height: 70px;
+  }
+  
+  .youtube {
+    text-decoration: none;
+  }
+  .search-form {
+    border-radius: 30px 0 0 30px;
+  }
+  
+  .input-group-btn {
+    max-width: 90px;
+  }
+  
+  .search-btn {
+    color: rgb(26, 26, 36);
```

Comment:
Great job describing the changes well but please consider always clarifying the **Reasons** for a change means describing clearly to the user why the change is necessary or being asked.

## Example of well explained request:--

>  Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible